lib/griddler/email.rb

```diff
@@ -3,7 +3,7 @@
 module Griddler
   class Email
     include ActionView::Helpers::SanitizeHelper
-    attr_reader :to, :from, :cc, :bcc, :subject, :raw_body, :raw_text, :raw_html, :headers, :raw_headers, :attachments, :content_ids, :envelope
```

Comment:
@vitor-mda Thanks for the MR. Is that related to fixing the Chinese/Korean characters issues with received emails? If that's the case, have we made sure that the charset we receive from the gem is actually the correct one? Also, how would that be used in pipefy-core to fix the issue?

Reply:
@md-hamed Yup, we're working on a fix to those emails which are saved without proper encoding handling.

From what I've gathered from logs, the text encoding seems to be properly set:
![image](https://user-images.githubusercontent.com/74683300/186922171-338dc4ca-6486-4c8a-8f37-8772a9663d32.png)

By having this info come in the email object, a simple `.encode('utf-8', source_charset)` would do the trick:
![image](https://user-images.githubusercontent.com/74683300/186923068-95c2f7d6-b777-48e5-90c5-c5a720779762.png)

The way encoding is currently handled depends on matching HTML content (which most problematic emails don't have) with a RegEx in order to extract the encoding.

Reply:
The intent is to change behavior the least possible to avoid breaking stuff, we're just gonna change how the charset is gotten so we can handle these cases properly.

Reply:
@vitor-mda thanks for the explanation. It's good to know that we already have the source encoding sat; as the last time I checked, emails received from Outlook didn't have it!